cdap-ui/app/features/dashboard/routes.js

```diff
@@ -32,17 +34,28 @@ angular.module(PKG.name+'.feature.dashboard')
       .state('dashboard.standard.cdap', {
         url: '/cdap',
         templateUrl: '/assets/features/dashboard/templates/staticdashboard.html',
-        controller: 'OpsCdapCtrl'
+        controller: 'OpsCdapCtrl',
+        ncyBreadcrumb: {
+          label: 'System',
+          parent: 'dashboard'
```

Comment:
```dashboard``` state is not something that the user would be viewing. Its a parent state which holds both system and user dashboards. This should idealy be 'home' or 'overview' right?